### PR TITLE
Better Parapens

### PR DESCRIPTION
**Pull request by Kaedwuff**
-Adds dextrotoxin, a ~magical~ futuristic chemical that causes paralysis in the arms and legs but leaves the rest of the body able to function.  Basically, you can talk but can't move or use the radio after a short delay.  
-This chemical is suited for abductions and rendering a victim helpless.  Now, you can roleplay with your captor instead of staring gormlessly at them for 10 minutes in silence. It also lasts half as long as zombie powder did.
-Replaces zombie powder with dextrotoxin as the implemented chemical in all related antagonists items.
-Does not remove zombie powder, which still exists as a fake death chemical.

Feedback: https://forums.aurorastation.org/topic/13167-better-parapens/

### CODE REVIEW

**Comment by Sindorman** [comment]
1. this PR is not what thread OP and other people requested. They requested removal of parapen replace it with 4 different special pens.

2. This PR doesn't cover all radios, what about intercomms and mech radio and Borg radio, etc.

**Comment by Kaedwuff** [comment]
> 1. this PR is not what thread OP and other people requested. They requested removal of parapen replace it with 4 different special pens.

There were other people in that thread - like me - that said no, don't remove them, they're fine.  I agree with them and don't particularly care about the people who disagree, just like the person who made a PR adding those four new options doesn't seem to want to listen to me and the others who want to keep parapens in.

I don't see any reason why we can't coexist and add our respective ideas. I'm simply addressing the subject that came up in the thread about it being hard to roleplay with people while being parapenned.  Hence, this having its own project thread separate from the original suggestion thread.

> 2. This PR doesn't cover all radios, what about intercomms and mech radio and Borg radio, etc.

Good point, I wasn't aware there is a difference. I'll fix that.

**Comment by AmoryBlaine** [comment]
It needs a better name. Something that isn't a joke, for the sake of a joke.

**Comment by Kaedwuff** [comment]
Done.  It is now called dextrotoxin

**Comment by Kaedwuff** [comment]
!review

**Comment by Kaedwuff** [comment]
The PR covers intercomms just fine, I checked.

File: code/modules/mob/living/living_defines.dm [hunk 1]
```diff
@@ -40,6 +40,7 @@
 	var/tod = null // Time of death
 	var/update_slimes = 1
 	var/silent = null 		// Can't talk. Value goes down every life proc.
+	var/noradio = null // Softer form of silence - can't use radios, but can talk.
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/living_defines.dm]: I'm sure this can be handled in a different way. Like a CE_ effect.

Comment (Kaedwuff) [reply code/modules/mob/living/living_defines.dm]: Like this?

**Comment by Sindorman** [comment]
> > 1. this PR is not what thread OP and other people requested. They requested removal of parapen replace it with 4 different special pens.
> 
> There were other people in that thread - like me - that said no, don't remove them, they're fine. I agree with them and don't particularly care about the people who disagree, just like the person who made a PR adding those four new options doesn't seem to want to listen to me and the others who want to keep parapens in.
> 
> I don't see any reason why we can't coexist and add our respective ideas without obviating the fun of others. I'm simply addressing the subject that came up in the thread about it being hard to roleplay with people while being parapenned. Hence, this having its own project thread separate from the original suggestion thread.

I did not notice this had its own thread. I would be okay with both rebalanced parapen and special pens kits both in the game.

File: code/modules/mob/living/living_defines.dm (hunk 1 above)
Comment (NonQueueingMatt) [reply code/modules/mob/living/living_defines.dm]: yep

**Comment by Arrow768** [comment]
Related to #7425

**Pull request merged**